c-dependencies/js-compute-runtime/builtins/fastly.cpp

```diff
@@ -16,7 +16,7 @@
 
 namespace builtins {
 
-bool Fastly::debug_logging_enabled = false;
+bool Fastly::debug_logging_enabled = true;
```

Comment:
Was this potentially left-over from a debugging session?

```suggestion
bool Fastly::debug_logging_enabled = false;
```

Reply:
Thanks for catching that.